charts/mothership/templates/bundler.yaml

```diff
@@ -0,0 +1,80 @@
+{{ range $idx := until (int $.Values.bundler.count) }}
+{{ $index := add $idx 1 }}
```

Comment:
just wondering, are there any reason to have `count` value instead of boolean `enabled` flag? would multiple bundler instances be needed?

Reply:
It is not necessary in the test environment, but it may be needed for high availability if the number of users increases. For now, handling it with an enabled flag is fine.

Reply:
I agree that it should be considered for HA, but there would be some points to be checked for running multiple instances for now.
for example, if it's for load balancing, i think upstream geth rpc also should be balanced, instead of fixed node-1:8545 (it might be resolved with optimism proxyd(rpc-proxy)), and I'm not sure running multiple bundlers with same private key is safe🤔 (eg. corrupt bundler tx nonces)

Reply:
> I agree that it should be considered for HA, but there would be some points to be checked for running multiple instances for now. for example, if it's for load balancing, i think upstream geth rpc also should be balanced, instead of fixed node-1:8545 (it might be resolved with optimism proxyd(rpc-proxy)), and I'm not sure running multiple bundlers with same private key is safe🤔 (eg. corrupt bundler tx nonces)

You're right.
There seem to be additional considerations related to `High Availability`. So, it might be better to address HA later.